src/objectid.ts

```diff
@@ -21,7 +21,7 @@ export interface ObjectIdExtended {
   $oid: string;
 }
 
-const kId = Symbol('id');
+const kId = Symbol.for('id');
```

Comment:
`Symbol.for` uses a global namespace for values, so you probably want to make the key here more specific than just `'id'`

Reply:
Good point, any preference according to naming conventions?  I could use `@@bson.objectid.id`...

Reply:
I picked the second one ;-)  If it's not OK, can change...

Reply:
I think we'd probably prefer the former `@@bson.objectid.id` since BSON is technically it's own library and doesn't necessarily need to be used the MongoDB.

Reply:
Changed to `@@bson.objectid.id`.